release/rllib_tests/learning_tests/yaml_files/a-e/ddpg-hopperbulletenv-v0.yaml

```diff
@@ -31,6 +31,7 @@ ddpg-hopperbulletenv-v0:
           prioritized_replay_beta: 0.4
           prioritized_replay_eps: 0.000001
           learning_starts: 500
+          worker_side_prioritization: false
```

Comment:
what does this do? Does this solve the error in the test?
was the parameter incorrectly placed in the yaml or something?

Reply:
yeah, this param was depreacated, error=True, so must fix this.
it's crashing right now.

Reply:
gotcha

Reply:
Yeah, was moved into `replay_buffer_config`.